campaigns/csp/guides/bandwidth10dlcBrandCspApiGuide.md

```diff
@@ -620,14 +623,15 @@ _Note_: Non-editable fields will be ignored and will not be updated.
 |:---------------------------|:----------|:---------|:-----------------------------------------------------|
 | `DisplayName`              | Yes       | Yes      | Display or marketing name of the brand. Max 100 characters   |
 | `Website`                  | No        | Yes      | Brand website URL. Max Length 100 characters  |
-| `Street`                   | No        | Yes      | street name. Max Length 100 characters |
-| `City`                     | No        | Yes      | City name. Max Length 100 characters  |
-| `State`                    | No        | Yes      | State name. Must be 2 letters code for United States  |
-| `PostalCode`               | No        | Yes      | Postal codes. Use 5 digit zipcode for United States  |
+| `Street`                   | Yes       | Yes      | street name. Max Length 100 characters |
+| `City`                     | Yes       | Yes      | City name. Max Length 100 characters  |
+| `State`                    | Yes       | Yes      | State name. Must be 2 letters code for United States  |
+| `PostalCode`               | Yes       | Yes      | Postal codes. Use 5 digit zipcode for United States  |
 | `Country`                  | Yes       | Yes      | ISO2 2 characters country code. Example: US - United States   |
 | `Email`                    | Yes       | Yes      | Valid email address of brand support contact. Max 100 characters   |
-| `Phone`                    | No        | Yes      | Valid phone number in e.164 international format '+18009999999' |
-| `Vertical`                 | Yes       | Yes      | Enum value describing vertical or industry segment of the brand   |
+| `Phone`                    | Yes       | Yes      | Valid phone number in e.164 international format '+18009999999' |
+| `Vertical`                 | Yes (Not required for Sole_Proprietor)       | Yes      | Enum value describing vertical or industry segment of the brand   |
+| `BrandRelationship`        | Yes       | Yes      | Enum value describing the relationship with your Account. Supported values: 'BASIC_ACCOUNT', 'SMALL_ACCOUNT', 'MEDIUM_ACCOUNT', 'LARGE_ACCOUNT', 'KEY_ACCOUNT' |
```

Comment:
I don't think BrandRelationship is editable on the PUT

Reply:
@jhough07 Thanks, I'll update. Just wanted to check: But it is still part of the request and response but not editable, right?

Reply:
A follow-up question. What if the reseller has to update `brandRelationship` for a customer brand after a few months?

Reply:
I was incorrect, it was editable